invenio_rdm_migrator/streams/models/records.py

```diff
@@ -88,6 +91,21 @@ class RDMRecordFile(Model):
     object_version_id: Mapped[UUID]
 
 
+class RDMRecordMediaFile(Model):
+    """RDM Record Media File dataclass model."""
+
+    __tablename__: InitVar[str] = "rdm_records_media_files"
+
+    id: Mapped[UUID] = mapped_column(primary_key=True)
+    json: Mapped[dict] = mapped_column(JSONB())
```

Comment:
Since it is defined in the base class, there might be many previous occurences of this.
```suggestion
    json: Mapped[dict]
```